pkg/util/attime/attime.go

```diff
@@ -24,8 +24,19 @@ func Parse(s string) time.Time {
 			return t
 		}
 
+		// Unix time.
 		v, _ := strconv.Atoi(s)
-		return time.Unix(int64(v), 0)
+		switch len(s) {
+		default:
+			// Seconds.
+			return time.Unix(int64(v), 0)
```

Comment:
We might need to revisit this logic around November 20, 2286.

Reply:
I'll set a reminder!